hack/run-test-upgrade-e2e.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+source hack/common.sh
+
+NO_COLOR=""
+if ! which tput &> /dev/null 2>&1 || [[ $(tput -T$TERM colors) -lt 8 ]]; then
+  echo "Terminal does not seem to support colored output, disabling it"
+  NO_COLOR="-ginkgo.no-color"
+fi
+
+setupreport
+
+# Run upgrade test suite
+echo "Running NRO upgrade test suite"
+if ! "${BIN_DIR}"/e2e-nrop-upgrade.test ${NO_COLOR} --ginkgo.v --ginkgo.timeout=1h --ginkgo.fail-fast --ginkgo.junit-report=${REPORT_DIR}/install.xml --ginkgo.focus='\[Upgrade\]'; then
```

Comment:
we should use labels (not tags/focus) but we can and probably do a sweeping change later

Reply:
Do we have ginkgo labels in NROP in general?
if not we must add some

Reply:
yes, we have some and we are trending (slowly) towards use only them

Reply:
Will change to use labels.